pennylane_qsharp/device.py

```diff
@@ -54,7 +57,7 @@
         // reset all qubits
         ResetAll(q);
     }}
-    return BoolArrFromResultArr(resultArray);
+    return ResultArrayAsBoolArray(resultArray);
```

Comment:
Weird, did they change this function?

Reply:
Yes, running the previous one would give me deprecation warnings